core/src/androidTest/kotlin/br/com/concrete/yosef/property/spacing/MarginCommandTest.kt

```diff
@@ -0,0 +1,67 @@
+package br.com.concrete.yosef.property.spacing
+
+import android.support.test.InstrumentationRegistry
+import android.support.test.runner.AndroidJUnit4
+import android.widget.LinearLayout
+import android.widget.TextView
+import br.com.concrete.yosef.api.property.spacing.MarginPropertyCommand
+import br.com.concrete.yosef.api.property.spacing.MarginPropertyCommand.Companion.MARGIN
+import br.com.concrete.yosef.entity.DynamicProperty
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.ExpectedException
+import org.junit.runner.RunWith
+import kotlin.math.roundToInt
+
+@RunWith(AndroidJUnit4::class)
+class MarginCommandTest {
+
+    @Rule
+    @JvmField
+    val exceptionRule: ExpectedException = ExpectedException.none()
```

Comment:
the exceptionRule is not being used in this class

Reply:
done